code/game/objects/effects/contraband.dm

```diff
@@ -635,4 +635,63 @@
 	// And yet people still believe...
 	icon_state = "poster37_legit"
 
+//Posters ported from Beestation, and inspired by Aspev's SS13 "Safety Moth" artworks!
+//Original PR at https://github.com/BeeStation/BeeStation-Hornet/pull/1747 (Also pull/1982)
+/obj/structure/sign/poster/official/moth_poison
+	name = "Safety Moth - Poisoning"
+	desc = "This informational poster uses Safety Moth(TM) to tell the viewer not to poison the station donuts. It's signed by 'AspEv'."
```

Comment:
Actually I'm a bit weary of including the guy's username on these IC portraits

Also you should replace (TM) with just ™

Reply:
TM fixed, gonna hold off on the signature change until theres a tad more input on it

Reply:
AspEv asked for direct attribution when we added these to bee.

Reply:
![spebbv](https://user-images.githubusercontent.com/76465278/141369863-d6d90404-6e5c-499b-b62b-da45938fad9d.jpg)
asked 'em myself, they said its fine not having it directly in the descriptions; working out rn if they'd prefer A.E. or just to do it in code comments

Reply:
aaand they said just code comments are fine 😎

Reply:
done and committed; credits to AspEv are now in the code comments